lib/dicom_net/association.ex

```diff
@@ -242,53 +296,57 @@ defmodule DicomNet.Association do
         AffectedSOPClassUID: DataElement.value(asci_de),
         CommandField: 0x8020,
         MessageIDBeingRespondedTo: DataElement.value(mid_de),
-        CommandDataSetType: 0x0001, # This field shall be set to the value of 0101H (Null) if no Data Set is present; any other value indicates a Data Set is included in the Message.
-        Status: status # 0x0000 Success, 0xff00 Pending
+        # This field shall be set to the value of 0101H (Null) if no Data Set is present; any other value indicates a Data Set is included in the Message.
+        CommandDataSetType: @dataset_present,
+        # 0x0000 Success, 0xff00 Pending
+        Status: status
       )
 
     Dicom.BinaryFormat.serialize_command_data_set(response)
-      |> Pdu.new_data_pdu(presentation_context_id) 
-      |> Pdu.serialize()
+    |> Pdu.new_data_pdu(presentation_context_id, type: :command_last_fragment)
```

Comment:
Perfect! So much easier to read the code now, and directly see that command dataset is used with command PDU type.